src/themes/eLife/styles.css

```diff
@@ -12,6 +12,16 @@
   --bg-color: #fff;
 }
 
+@custom-selector :--cite-group [itemtype='https://stencila.github.io/schema/CiteGroup'];
+@custom-selector :--cite [itemtype='https://stencila.github.io/schema/Cite'];
+@custom-selector :--creative-work [itemtype='https://schema.org/CreativeWork'];
+@custom-selector :--title [itemprop='title'];
+@custom-selector :--authors [itemprop='authors'];
+@custom-selector :--person [itemtype='https://schema.org/Person'];
```

Comment:
Would it be useful to use the same naming convention for custom selectors as for types and properties: UpperCamelCase for types, and lowerCamelCase for properties.

```css
@custom-selector :--Person [itemtype='https://schema.org/Person'];
@custom-selector :--name [itemtype='https://schema.org/name'];
```

Reply:
Something to also consider is to add [Stylelint rule to discourage using `[itemtype/prop=*]`](https://stylelint.io/user-guide/rules/selector-attribute-operator-blacklist) selectors in the the CSS, and enforce just using the custom selectors.